### ISSUE TITLE
Command not found wsdlTojava

### ISSUE THREAD

**Issue by ashalitkin** [Grails-Plugin-Consortium/grails-cxf-client#71]
I have grails 3.2.2 and the latest cxf-client version (3.0.6). Even if I configure exactly as per instructions in docs, it still gives me an errror:
```
:~/workspace/git/inventory-microservices/inventory-core$ grails wsdlTojava
| Error Command not found wsdlTojava
Did you mean: wsdl2java or wsdl-to-java or stop?

```

Should I sue wsdl-to-java?

Also, I've tried wsdl-to-java but if the class that is specified as clientInterface doesn't exist it gives me an error saying that it can't be found. It's like a circle now - I can't generate classes without clientInterface but the clientInterface itself must be generated.

**Comment by ashalitkin** [comment]
It turned out that It's actually resolved in 3.0.7. Closing it.

**Comment by ashalitkin** [comment]
Also, as for wsdl-to-java - it's really a mistake in docs, going to create a pull request for it.
